commands/drawCommand.js

```diff
@@ -22,14 +25,101 @@ export const RARITIES = [
   "Gold Ultra Rare",
 ];
 
+const storingDraws = async (message) => {
+  const response = await axios.get(
+    "https://db.ygoprodeck.com/api/v7/randomcard.php"
+  );
+  let highestRarityIndex = 0;
+  const card = response.data.card_sets;
+
+  card.forEach((element) => {
+    const cardSetRarity = element.set_rarity;
+    const cardSetRarityIndex = RARITIES.indexOf(cardSetRarity);
+    if (highestRarityIndex < cardSetRarityIndex) {
+      highestRarityIndex = cardSetRarityIndex;
+    }
+  });
```

Comment:
if you wanted, now that you've mastered some of the other ES6 array functions, you could get rid of the `let` here using one of the tougher ones: [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Using array reduce, you can turn these 10 lines into 3 or 4 I think, and get rid of many variables